src/main/java/com/bgsoftware/superiorskyblock/island/bank/SIslandBank.java

```diff
@@ -116,16 +116,19 @@ public BankTransaction depositAdminMoney(CommandSender commandSender, BigDecimal
         SuperiorSkyblockPlugin.debug("Action: Deposit Money, Island: " + island.getOwner().getName() + ", Player: " + commandSender.getName() + ", Money: " + amount);
 
         UUID senderUUID = commandSender instanceof Player ? ((Player) commandSender).getUniqueId() : null;
+        boolean isConsole = !(commandSender instanceof Player);
 
```

Comment:
A better solution would be to make another method that takes as a parameter whether to save the transaction or not, instead of not saving it when console makes the payment. Also, addTransaction takes a similar parameter to whether to save the transaction to database or not, and it should be used instead.

Reply:
So I should make the same method but with an extra parameter?

Reply:
> 

Yes, and make the original method to call the new one with a default parameter.